### PR TITLE
Spanish Translation and English Scroll Docs Fixes

### LINKED ISSUE #81: [Enhancement]: Translation to Spanish and English fixes in Docs
### What would you like us to improve?

Hello, I have completed a comprehensive Spanish translation of the Scroll's Docs. This translation encompasses not only the content but also includes the translation of buttons and sidebars.

During the translation process, I identified broken links and "What's Next" buttons in the English Docs version, and I have successfully rectified these issues.

### How important it is?

The broken links and the "what's next" button are currently live on the website, so it's crucial to address and fix these issues as promptly as possible.

We believe that Scroll holds significant potential within Spanish-speaking communities. As part of our commitment to reducing language barriers, we have undertaken the task of translating all your documentation. This initiative will create opportunities to welcome more non-English speakers into the Scroll Ecosystem who are eager to contribute and engage.

### PR DESCRIPTION
**Pull request by ManuGodzilla**
## Closing issues

closes #81 

## Description
Translation of english Scroll documents into spanish and correction of errors in english Scroll pages.

## Changes

- Created the ` \src\content\docs\es` Directory.
- Created the `\public\locales\es\translation.json` Directory
- We translated the Whatsnext button and the right side title such as "On this page", "More", "Edit this Page", "Join into our Community" and "Feedback"
- During the Translation, we found a bug at `src\components\RightSidebar\TableOfContents\index.tsx`, the right side bar title " On This Page" it disappear, even when we follow the exact command to implemented into spanish. We think it's a rendering code error out of this scope that should be addressed by other PR.

### CODE REVIEW

**Comment by dghelm** [comment]
> During the Translation, we found a bug at src\components\RightSidebar\TableOfContents\index.tsx, the right side bar title " On This Page" it disappear, even when we follow the exact command to implemented into spanish. We think it's a rendering code error out of this scope that should be addressed by other PR.

@zzq0826 might have an idea for what's happening here.

**Comment by zzq0826** [comment]
> > During the Translation, we found a bug at src\components\RightSidebar\TableOfContents\index.tsx, the right side bar title " On This Page" it disappear, even when we follow the exact command to implemented into spanish. We think it's a rendering code error out of this scope that should be addressed by other PR.
> 
> @zzq0826 might have an idea for what's happening here.

@dghelm @ManuGodzilla Fixed this issue [here](https://github.com/scroll-tech/scroll-documentation/pull/85/files). There was a problem with the CSR earlier. (Need to rebase the `develop` branch to make it work here).

File: src/content/docs/es/developers/index.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+lang: "es"
+permalink: "developers/"
+excerpt: "Construir en Scroll se siente igual que en Ethereum, y puedes llevar contigo tus herramientas y contratos favoritos."
+whatsnext: { "Inicio rápido para desarrolladores": "/es/developers/developer-quickstart" }
+---
+
+import Aside from "../../../../components/Aside.astro"
+import ToggleElement from "../../../../components/ToggleElement.astro"
+
+**Bienvenido a la documentación para desarrolladores de Scroll!**
```
Comment (Turupawn) [inline src/content/docs/es/developers/index.mdx]: Corrección: agregar ¡
¡Bienvenidos*

File: src/content/docs/es/developers/index.mdx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    Tras el merge, los bloques de Ethereum se confirman de forma fiable cada 12 segundos. Los bloques de Scroll se mintean cada 3 segundos, y
+    en favor de las operaciones de menor riesgo, se puede suponer que las transacciones son definitivas una vez incluidas en un bloque. 
+    Esto abre nuevas posibilidades para la interacción en la cadena en aplicaciones sociales y de juegos.
+  </p>
+</ToggleElement>
+<ToggleElement anchor="alignment">
+  <div slot="title">Alineamiento - Scroll se basa en la visión de Ethereum.</div>
+  <p>
+    Scroll se basa en la visión de Ethereum. Nuestra filosofía es construir Ethereum, no fragmentarlo. La descentralización,
+    el ideal permissionless, la resistencia a la censura, y la propiedad de la comunidad son el núcleo de lo que hacemos y
```
Comment (Turupawn) [inline src/content/docs/es/developers/index.mdx]: Qué tal suena?
"empoderamiento de la comunidad"
en vez de "la propiedad de la comunidad" (eng: "community ownership")

File: src/content/docs/es/developers/index.mdx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    en favor de las operaciones de menor riesgo, se puede suponer que las transacciones son definitivas una vez incluidas en un bloque. 
+    Esto abre nuevas posibilidades para la interacción en la cadena en aplicaciones sociales y de juegos.
+  </p>
+</ToggleElement>
+<ToggleElement anchor="alignment">
+  <div slot="title">Alineamiento - Scroll se basa en la visión de Ethereum.</div>
+  <p>
+    Scroll se basa en la visión de Ethereum. Nuestra filosofía es construir Ethereum, no fragmentarlo. La descentralización,
+    el ideal permissionless, la resistencia a la censura, y la propiedad de la comunidad son el núcleo de lo que hacemos y 
+    la hoja de ruta que estamos construyendo. Creemos en el software de código abierto y colaboramos estrechamente con el
```
Comment (Turupawn) [inline src/content/docs/es/developers/index.mdx]: Qué tal suenan?
"el plan" o simplemente "el roadmap"
en vez de "la hoja de ruta"

File: src/content/docs/es/developers/index.mdx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  </p>
+  <p>
+    También trabajamos con DAOs de gobernanza y otros protocolos de código abierto para asegurarnos de que, a medida que se despliegan las aplicaciones,
+    trabajamos para aumentar su impacto, ya sea en bienes públicos, infraestructura básica o la próxima generación de casos de uso de zero-knowledge.
+  </p>
+</ToggleElement>
+<ToggleElement anchor="community">
+  <div slot="title">Comunidad - Scroll reúne a usuarios y builders.</div>
+  <p>
+    Conocemos los retos de construir en abierto y conseguir la participación de los usuarios antes del lanzamiento de la mainnet.
```
Comment (Turupawn) [inline src/content/docs/es/developers/index.mdx]: Qué tal suena?
"abiertamente"
en vez de "en abierto"

**Review by Turupawn — COMMENTED** [review]
Observaciones sobre "Contruyendo en Scroll"

**Comment by Turupawn** [comment]
Closing this and moving it to #95 due to conflicts with this branch

**Pull request closed without merging**